modules/ROOT/pages/vm/vm-reference.adoc

```diff
@@ -91,11 +90,11 @@ The flow in which the content is published cannot contain a Listener (`<vm:liste
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  A TimeUnit which qualifies the #timeoutUnit |  SECONDS | {nbsp}
-| Target Variable a| String |  The name of a variable in which the operation's output is placed. |  | {nbsp}
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression will be stored in the target variable. |  #[payload] | {nbsp}
+** DAYS |  A TimeUnit which qualifies the #timeoutUnit |  SECONDS | 
+| Target Variable a| String |  The name of a variable in which the operation's output is placed. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression will be stored in the target variable. |  #[payload] | 
```

Comment:
I have changed this wording to "An expression that evaluates against the operations's output. The outcome of the expression is stored in the target variable."